documentation/src/main/asciidoc/user_guide/chapter-23-Persistence.adoc

```diff
@@ -57,7 +57,10 @@ A provider might choose to only implement a subset of these interfaces:
 
 * Not implementing the  link:http://docs.jboss.org/infinispan/{infinispanversion}/apidocs/org/infinispan/persistence/spi/AdvancedCacheWriter.html[AdvancedCacheWriter] makes the given writer not usable for purging expired entries or clear
 
-* Not implementing  the link:http://docs.jboss.org/infinispan/{infinispanversion}/apidocs/org/infinispan/persistence/spi/AdvancedCacheWriter.html[AdvancedCacheLoader] makes the information stored in the given loader not used for preloading, nor for the map/reduce iteration
+* Not implementing  the link:http://docs.jboss.org/infinispan/{infinispanversion}/apidocs/org/infinispan/persistence/spi/AdvancedCacheWriter.html[AdvancedCacheLoader]
```

Comment:
This sentence was yucky before (I don't like the double negation). Better write it as:
If a loader does not implement the link:http://docs.jboss.org/infinispan/{infinispanversion}/apidocs/org/infinispan/persistence/spi/AdvancedCacheWriter.html[AdvancedCacheWriter] inteface, then it will not participate in preloading nor in cache iteration (required also for streams).